redis/files/redis-2.4.conf.jinja

```diff
@@ -106,423 +41,86 @@ save 300 10
 save 60 10000
```

Comment:
You are right in that these simply are the default, but I'd either omit them completely or keep these defaults in map.jinja.